### PR TITLE
add new folder and files , add unit test

### PR DESCRIPTION
**Pull request by AgnieszkaKurek**

### CODE REVIEW

File: functions/exercise14/scriptSpec.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getAmountOfCoins*/
+
+describe("Getting  amount of coins ", function () {
+    it("in the amount of 5 are 5", function () {
+        var amountOfCoins = getAmountOfCoins (5);
+        expect(amountOfCoins).toEqual([5]);
+    });
+    it("in the amount of 45 are 25, 10, 10, 1", function () {
```
Comment (Krzysiek102) [inline functions/exercise14/scriptSpec.js]: Incorrect calculation

File: functions/exercise14/scriptSpec.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getAmountOfCoins*/
+
+describe("Getting  amount of coins ", function () {
+    it("in the amount of 5 are 5", function () {
+        var amountOfCoins = getAmountOfCoins (5);
```
Comment (Krzysiek102) [inline functions/exercise14/scriptSpec.js]: Why do you have space there?

File: functions/exercise14/scriptSpec.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        expect(amountOfCoins).toEqual([5]);
+    });
+    it("in the amount of 45 are 25, 10, 10, 1", function () {
+        var amountOfCoins = getAmountOfCoins (45);
```
Comment (Krzysiek102) [inline functions/exercise14/scriptSpec.js]: Why do you have space there?

**Review by Krzysiek102 — CHANGES_REQUESTED** [review]
Please review your changes on your own before making a pull request. You constantly make silly mistakes which can be easily eliminated by revising it on your own.

File: functions/exercise14/scriptSpec.js (hunk 1 above)
Comment (AgnieszkaKurek) [inline functions/exercise14/scriptSpec.js]: 14. Write a JavaScript function to convert an amount to coins. Go to the editor
Sample function : amountTocoins(46, [25, 10, 5, 2, 1])
Here 46 is the amount. and 25, 10, 5, 2, 1 are coins. 
Output : 25, 10, 10, 1

**Review by Krzysiek102 — APPROVED** [review]

**Pull request merged**